controllers/argocd/status.go

```diff
@@ -93,6 +98,21 @@ func (r *ReconcileArgoCD) reconcileStatusDex(cr *argoprojv1a1.ArgoCD) error {
 	return nil
 }
 
+// reconcileStatusSsoConfig will ensure that the SsoConfig status is updated for the given ArgoCD.
+func (r *ReconcileArgoCD) reconcileStatusSsoConfig(cr *argoprojv1a1.ArgoCD) error {
+	status := "Unknown"
+
+	if cr.Spec.SSO != nil && !reflect.DeepEqual(cr.Spec.Dex, argoprojv1a1.ArgoCDDexSpec{}) {
+		status = "Only can install one SSO provider, please uninstall Dex or keycloak"
```

Comment:
Can we rephrase this to say something like "Installation of multiple SSO providers simultaneously is not permitted. Please choose a single provider"

Reply:
I would keep the ssoConfig status consistent with other statuses.  That is, I would stick with something like: Unknown/Success/Failed.

Reply:
Thanks for the recommendation. This is implemented as `Unknown/Success/Failed`.